cmds/info.c

```diff
@@ -853,6 +853,116 @@ static int read_uftrace_version(void *arg)
 	return 0;
 }
 
+void write_header(int fd, struct uftrace_file_header *hdr)
+{
+	dprintf(fd, "magic:%s\n", hdr->magic);
+	dprintf(fd, "version:%u\n", hdr->version);
+	dprintf(fd, "header_size:%u\n", hdr->header_size);
+	dprintf(fd, "endian:%u\n", hdr->endian);
+	dprintf(fd, "elf_class:%u\n", hdr->elf_class);
+	dprintf(fd, "feat_mask:%lu\n", hdr->feat_mask);
+	dprintf(fd, "info_mask:%lu\n", hdr->info_mask);
```

Comment:
In 32-bit architecture, the size of `%lu` is not 64-bit and compiler complains about it.
```
$ ./configure --arch=i386
$ make -j
      ...
cmds/info.c: In function ‘write_header’:
cmds/info.c:863:27: warning: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 3 has type ‘uint64_t’ {aka ‘long long unsigned int’} [-Wformat=]
  863 |  dprintf(fd, "feat_mask:%lu\n", hdr->feat_mask);
      |                         ~~^     ~~~~~~~~~~~~~~
      |                           |        |
      |                           |        uint64_t {aka long long unsigned int}
      |                           long unsigned int
      |                         %llu
cmds/info.c:864:27: warning: format ‘%lu’ expects argument of type ‘long unsigned int’, but argument 3 has type ‘uint64_t’ {aka ‘long long unsigned int’} [-Wformat=]
  864 |  dprintf(fd, "info_mask:%lu\n", hdr->info_mask);
      |                         ~~^     ~~~~~~~~~~~~~~
      |                           |        |
      |                           |        uint64_t {aka long long unsigned int}
      |                           long unsigned int
      |                         %llu
```
We should explicitly set it to 64-bit with `"%"PRIu64` as follows.
```suggestion
	dprintf(fd, "feat_mask:%"PRIu64"\n", hdr->feat_mask);
	dprintf(fd, "info_mask:%"PRIu64"\n", hdr->info_mask);
```